### PR TITLE
format & lint – `cg/ops/tests*.py`

### PR DESCRIPTION
**Pull request by jGaboardi**
xref https://github.com/pysal/libpysal/issues/589

This PR builds on top of #610 (should review and merge that one first) and formats & lints `cg/ops/tests*.py`

### CODE REVIEW

File: libpysal/cg/ops/tests/test_accessors.py [hunk 1]
```diff
@@ -211,7 +211,7 @@ def test_parts(self):
         with pytest.raises(AttributeError):
             to_test.parts(self.points)
 
-        line_parts = to_test.parts(self.lines)
+        to_test.parts(self.lines)
```
Comment (martinfleis) [inline libpysal/cg/ops/tests/test_accessors.py:214]: I am not super excited about changes like this. `parts` return something. We then don't use it but wouldn't a better solution be `_ = to_test.parts(self.lines)`? To indicate that this indeed returns something but we intentionally ignore it?

File: libpysal/cg/ops/tests/test_accessors.py [hunk 2]
```diff
@@ -275,42 +275,9 @@ def test_parts(self):
                 ]
             ],
         ]
-        line_answers = [
```
Comment (martinfleis) [inline libpysal/cg/ops/tests/test_accessors.py:278]: huh? Maybe the `line_parts` above should have been tested against this.

File: libpysal/cg/ops/tests/test_accessors.py (hunk 1 above)
Comment (jGaboardi) [reply libpysal/cg/ops/tests/test_accessors.py:214]: Seems to be another auto fix snafu that went under the radar (including your comment below). I'll look into it.

Comment (jGaboardi) [reply libpysal/cg/ops/tests/test_accessors.py]: @martinfleis  So, what it seems like is the `line_parts` vs. `line_answers` was never actually being tested, which led to `ruff` determining it was unused code marked for removal. I popped in a testing chunk for `line_parts` vs. `line_answers` and all seems appropriate now.

File: libpysal/cg/ops/tests/test_accessors.py (hunk 2 above)
Comment (jGaboardi) [reply libpysal/cg/ops/tests/test_accessors.py:278]: See [above](https://github.com/pysal/libpysal/pull/611#discussion_r1376189795).

**Review by martinfleis — APPROVED** [review]

**Pull request merged**